mne/decoding/time_delaying_ridge.py

```diff
@@ -6,6 +6,7 @@
 # License: BSD (3-clause)
 
 import numpy as np
+from scipy.signal import fftconvolve
```

Comment:
This needs to be nested inside `predict` (we lazy load scipy submodules because they take some time, and we want to keep `import mne` as lightweight as possible)